third_party/move/move-compiler-v2/src/options.rs

```diff
@@ -56,31 +69,52 @@ pub struct Options {
         num_args = 0..
     )]
     pub experiments: Vec<String>,
+
     /// A transient cache for memoization of experiment checks.
     #[clap(skip)]
     pub experiment_cache: RefCell<BTreeMap<String, bool>>,
+
     /// Sources to compile (positional arg, therefore last).
     /// Each source should be a path to either (1) a Move file or (2) a directory containing Move
     /// files, all to be compiled (e.g., not the root directory of a package---which contains
     /// Move.toml---but a specific subdirectorysuch as `sources`, `scripts`, and/or `tests`,
     /// depending on compilation mode).
     pub sources: Vec<String>,
+
     /// Dependencies to compile but not treat as a test/docgen/warning/prover target.
     /// Each source_dep should be a path to either (1) a Move file or (2) a directory containing
     /// Move files, all to be compiled (e.g., not the root directory of a package---which contains
     /// Move.toml---but a specific subdirectorysuch as `sources`).
     #[clap(skip)]
     pub sources_deps: Vec<String>,
+
+    #[clap(long = cli::MOVE_COMPILER_WARN_OF_DEPRECATION_USE_FLAG,
+           default_value=bool_to_str(move_compiler_warn_of_deprecation_use_env_var()))]
+    pub warn_deprecated: bool,
```

Comment:
Just wondering how this relates to the title of the PR. Should you update the title?

Reply:
Fixed it.  Though it's the final commit message that really matters.